apps/system/test/unit/lockscreen/lockscreen_state_manager_test.js

```diff
@@ -155,7 +153,11 @@ suite('system/LockScreenStateManager', function() {
         return true;
       });
       subject.transfer({}).then(() => {
-        assert.isTrue(stubTransferOut.called, 'the method isn\'t called');
+        assert.isTrue(stubTransferTo.called,
+          'the |transferTo| method isn\'t called');
+        assert.isTrue(stubTransferOut.called,
+          'the |transferOut| method isn\'t called');
+        done();
       }).catch(done);
```

Comment:
nits: this is how I write unit tests that have promises:
```javascript
functionReturningPromise().then(() => {
  // assertions
}).then(done, done);
```

Basically the same than what you wrote but I think it's more readable.

As another nit, you can use `sinon.assert.called(spy)` instead of `assert.isTrue(spy.called)`. This is a lot easier to debug when this fails, because sinon gives you some more information. Especially if you use `sinon.assert.calledWith` it will give you how the function was called.